tools/accuracy_checker/accuracy_checker/metrics/classification.py

```diff
@@ -288,6 +293,10 @@ def evaluate(self, annotations, predictions):
         )
         if self.profiler:
             self.profiler.finish()
+
+        if self.pos_label is not None:
+            return f1_score[self.pos_label]
```

Comment:
metric meta names should be modified in appropriate way (contains only this class)

Reply:
Done